grisera_api/activity_execution/activity_execution_service_graphdb.py

```diff
@@ -42,20 +47,19 @@ def save_activity_execution(self, activity_execution: ActivityExecutionIn):
                 type(self.activity_service.get_activity(activity_execution.activity_id)) is not NotFoundByIdModel:
             self.graph_api_service.create_relationships(start_node=activity_execution_id,
                                                         end_node=activity_execution.activity_id,
-                                                        name="hasActivity")
+                                                        name="Activity")
 
         if activity_execution.arrangement_id is not None and \
                 type(self.arrangement_service.get_arrangement(activity_execution.arrangement_id)) \
                 is not NotFoundByIdModel:
-
             self.graph_api_service.create_relationships(start_node=activity_execution_id,
                                                         end_node=activity_execution.arrangement_id,
                                                         name="hasArrangement")
 
         activity_execution.activity_id = activity_execution.arrangement_id = None
         self.graph_api_service.create_properties(activity_execution_id, activity_execution)
 
-        return self.get_activity_execution(activity_execution_id)
+        return self.get_activity_execution(activity_execution_id, 0)
```

Comment:
`depth` is by default `0`, why do we need to explicitly give `0` value? However, in such a case it's a good idea to pass a named argument: `depth=0` for better readability.